docs/index.rst

```diff
@@ -18,3 +18,13 @@ Documentation :any:`index <genindex>` and :any:`Module <modindex>` index.
   dev_guide/index
   whatsnew/index
   about
+
+.. toctree::
+  :maxdepth: 2
+
+  guide/index
+  code_ref/index
+  generated/gallery/index
+  dev_guide_release/index
+  whatsnew/index
+  about
```

Comment:
Why was this added?

Reply:
I added this because I thought sphinx will need everything again for the toctree of release version. I just built it and checked, it's unnccesory and only `dev_guide_release/index` needs to be added after `dev_guide/index`. Fixed.